junction/proposals/management/commands/export_conf_proposals.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+import csv
+from os.path import join
+from settings.common import ROOT_DIR
+
+from django.shortcuts import get_object_or_404
+from django.core.exceptions import PermissionDenied
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+from junction.conferences.models import Conference
+from junction.proposals.models import (
+    Proposal,
+    ProposalSectionReviewerVoteValue,
+    )
+from junction.base.constants import (
+    ProposalReviewVote,
+    ProposalStatus,
+    ProposalReviewStatus,
+    )
+from junction.proposals.permissions import is_conference_moderator
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('--conference_slug', default=None, help='Enter the conference slug where to export reviewer votes from')
+
+        parser.add_argument('--user_id', default=None, help='Enter your user id')
+
+    def handle(self, *args, **options):
+
+        try:
+            conference = get_object_or_404(Conference, slug=self.options.get('conference_slug'))
+            user = User.objects.get(id=self.options.get('user_id'))
+
+            if not is_conference_moderator(user=user, conference=conference):
+                raise PermissionDenied
```

Comment:
Will it be a good idea to raise an exception in the command, I mean it's not a view? I think you could do stdout and return?

Reply:
Makes sense, will do that